src/compiler/diagnosticMessages.json

```diff
@@ -2466,5 +2466,9 @@
     "A constructor cannot contain a 'super' call when its class extends 'null'": {
       "category": "Error",
       "code": 17005
+    },
+    "Left hand side of '**' cannot be a simple unary expression. Consider parenthesize the expression.": {

```

Comment:
"Simple unary expression" is too ambiguous. There really doesn't seem to be a clear definition here. You need something like "non-incrementing unary operator". I'd also use the phrase "Consider enclosing the expression in parentheses."

Do we just use this error for recovery purposes, rather than just reporting an invalid token? You could try something like:

> Unary '{0}' operator not allowed in the left-hand side of an '\*\*' expression. Consider enclosing the expression in parentheses.

So you'd end up with:

> Unary 'delete' operator not allowed in the left-hand side of an '\*\*' expression. Consider enclosing the expression in parentheses.
> Unary '+' operator not allowed in the left-hand side of an '\*\*' expression. Consider enclosing the expression in parentheses.